### DISCUSSION TITLE
where is the code for PIXART-δ?

### DISCUSSION THREAD

**Discussion by zmy-08** [PixArt-alpha/PixArt-alpha#116] [Q&A] [answered]
Hello, sorry to bother you, where is the code for PIXART-δ? Why don't you see it, it's all PIXART-a

**Comment by lawrence-cj** [comment, marked as answer]
https://github.com/PixArt-alpha/PixArt-alpha#3-pixart--lcm--lcm-lora-training
and 
https://github.com/PixArt-alpha/PixArt-alpha#4-pixart--controlnet-training

**Reply by zmy-08** [reply]
Thank you very much!
